test/parallel/test-webcrypto-random.js

```diff
@@ -47,7 +47,7 @@ for (const ctor of intTypedConstructors) {
 }
 
 {
-  const buf = new Uint16Array(10);
+  const buf = Buffer.alloc(10);
   const before = Buffer.from(buf).toString('hex');
```

Comment:
If you want to change `buf` to be a `Buffer`, then `before` and `after` can just be `buf.toString('hex')` instead of using `Buffer.from(buf).toString('hex')`.